src/Slider.tsx

```diff
@@ -144,13 +227,13 @@ export const Slider: React.FC<PropsWithChildren<Settings>> = ({ children, hideNa
 
             sortSlides();
 
-            if (hideNavigationButtons === false) {
+            if (!hideNavigationButtons) {
```

Comment:
I personally prefer the way it was, is way easier to read

Reply:
You already used the ! notation on the currentWrapper checks, so figured this would be more consistent.

Reply:
Fair enough, I generally use === false when dealing with bools. But I suppose that's just personal preference and hard to capture in a linting rule.